provider/bitmovin/bitmovin.go

```diff
@@ -979,6 +987,32 @@ func (p *bitmovinProvider) addOutputStatusInfo(job *db.Job, status *provider.Job
 	return nil
 }
 
+func (p *bitmovinProvider) addSourceInfo(job *db.Job, status *provider.JobStatus) error {
+	encodingS := services.NewEncodingService(p.client)
+	resp, err := encodingS.ListStream(job.ProviderJobID, 0, 1)
+	if err != nil {
+		return err
+	}
+	if len(resp.Data.Result.Items) > 0 {
+		streamID := stringValue(resp.Data.Result.Items[0].ID)
+		streamInput, err := encodingS.RetrieveStreamInputData(job.ProviderJobID, streamID)
+		if err != nil {
+			return nil
```

Comment:
Why `nil`?

Reply:
Because I'm crazy 😁 

Good catch, I will fix it. Will also add a test. Thanks!